### PR TITLE
PR checklist to be ordered and have only one task per item

### PR DESCRIPTION
**Pull request by heikoklein**
## Change Summary

PR checklist has been ordered to reflect work.
Each checkbox only covers one task.

## Checklist

* [x] Tests pass locally
* [x] Tests for the changes exist where applicable
* [x] Start with a draft-PR
* [x] The pull request title is a good summary of the changes
* [x] PR is set to AeroTools and milestone
* [x] Documentation reflects the changes where applicable
* [x] Tests pass on CI
* [x] At least 1 reviewer is selected
* [x] My PR is ready to review

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -9,9 +9,14 @@
 
 ## Checklist
 
+## Checklist
+
+* [ ] Tests pass locally
```
Comment (lewisblake) [inline .github/PULL_REQUEST_TEMPLATE.md]: This is not order in which things are done though. I agree that we could split some of checklist items into separate checks but they still should follow the order in which a PR is written. The first thing is starting with a draft PR.  I would suggest the following:

* [ ] Start with a draft-PR
* [ ]  The pull request title is a good summary of the changes
* [ ] Documentation reflects the changes where applicable
* [ ] Tests pass locally
* [ ] Tests pass on CI
* [ ] At least one reviewer is selected
* [ ] My PR is read to review

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -9,9 +9,14 @@
...
+* [ ] Tests for the changes exist where applicable
 * [ ] Start with a draft-PR
 * [ ] The pull request title is a good summary of the changes
+* [ ] PR is set to AeroTools and milestone
```
Comment (lewisblake) [inline .github/PULL_REQUEST_TEMPLATE.md]: This contradicts having only one item per task. Also, is setting the project to AeroTools is strictly necessary? We could easily set a GitHub action automatically if the person opening the PR forgets to so similar to what is done with the Issues. If we want to include it I would prefer writing "PR is set to the AeroTools project and has a milestone".

**Review by lewisblake — CHANGES_REQUESTED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (heikoklein) [reply .github/PULL_REQUEST_TEMPLATE.md]: updated after discussion

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (heikoklein) [reply .github/PULL_REQUEST_TEMPLATE.md]: It's kind of the same task (Ensure PR is set to correct project-phase).

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (heikoklein) [reply .github/PULL_REQUEST_TEMPLATE.md]: * [ ] Start with a draft-PR
* [ ] The PR title is a good summary of the changes
* [ ] PR is set to AeroTools and a tentative milestone
* [ ] Documentation reflects the changes where applicable
* [ ] Tests for the changes exist where applicable
* [ ] Tests pass locally
* [ ] Tests pass on CI
* [ ] At least 1 reviewer is selected
* [ ] Make PR ready to review

**Review by lewisblake — APPROVED** [review]

**Pull request merged**